src/plugins/vis_default_editor/public/_default.scss

```diff
@@ -70,8 +70,7 @@
 
 .visEditor__visualization {
   display: flex;
-  flex-basis: 100%;
-  flex: 1;
+  flex: 1 1 auto;
```

Comment:
This fixes another bug:
the editors overflows a visualization on small screens, the image attached in PR description.

Reply:
Can you add a comment for this one as well so it doesn't get removed?